tour.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<title>Tour</title>
+</head>
+<body>
+<div class="header">
+	<div class="wrapper">
+		<div class="top-header">
+			<div class="navigation">
+				<ul class="navigation-buttons">
+					<li><a href="#">About</a></li>
+					<li><a href="#">Audio</a></li>
+					<li><a href="#">Video</a></li>
+					<li><a href="#">Gallery</a></li>
+					<li><a href="#">Tour Dates</a></li>
```

Comment:
this page is called "Tour dates" - so you need a special class for a link ("active" or "current")